source/_integrations/reolink.markdown

```diff
@@ -84,3 +84,4 @@ The following models are lacking the HTTP webserver API and can therfore not wor
 - The Reolink NVR only sends event-notifications if motion happens on the camera connected to the first (index "0") channel, therefore the binary sensors of all channels will only be updated when the first channel sees motion. Beta NVR firmware v3.0.0.211_23011204 fixes this issue, you can request beta firmware from reolink support, release firmware is expected in a few weeks.
 - Reolink doorbell presses only generate ONVIF event notifications when the doorbell is directly connected to your network. The doorbell visitor binary sensor will not work when connecting the Reolink doorbell to an NVR. Beta NVR firmware v3.0.0.211_23011204 fixes this issue, you can request beta firmware from reolink support, release firmware is expected in a few weeks.
 - Older firmware versions do not expose the necessary information the integration needs to function. Ensure the camera is updated to the [latest firmware](https://reolink.com/download-center/) prior to setting up the integration.
+- Reolink products can not push motion events to a HTTPS address (SSL), you need to configure a local HTTP address under `settings->system->network->Home Assistant URL->local network` in order for motion/people/visitor/vehicle detection to properly work.
```

Comment:
- Add a my link, so users can quickly go there.
- Users generally should not have a need to edit those settings. These are more of an exception in case the user has configured SSL in their Home Assistant (in that case, they cannot resolve it via the suggest change either). So, I'm kinda thinking this whole addition is currently not workable for an end-user.

../Frenck